### ISSUE TITLE
Export interactive crabviz page

### ISSUE THREAD

**Issue by naseer-nv** [chanhx/crabviz#32]
Thanks for the utility!
I want to export the crabviz page entirely as html (including the click functionality) as well as reference the links to the code in an online browser. Can you please consider this feature request?

**Comment by chanhx** [comment]
Good idea! We can bundle all related files of a call graph into a single html file to share that interactive call graph.

And there is a plan for the "jump to the code" feature, but I would prefer to do it just in the editor, because links to the code just work in the local environment.

I think we could have these features in v0.5.0 .

**Comment by naseer-nv** [comment]
Jump to the code in the case of an exported html can be done by providing a base URL pattern as a user provided workspace config setting - that URL can be an opengrok link or a GitHub link for example. Crabviz can then use the relative code path to the workspace root and that base pattern to generate the link.

**Comment by chanhx** [comment]
Hmm, that is beyond the project scope.

**Comment by Yehuda-Bergstein** [comment]
This would be great! is there currently a way to do this in vscode?

(The export of an HTML file for external interactivity in a browser)

**Comment by Yehuda-Bergstein** [comment]
This would be great! is there currently a way to do this in vscode?

**Comment by CsatiZoltan** [comment]
Hi @chanhx. Do you have any update on this?

**Comment by chanhx** [comment]
Hmm, yes and no. The feature development was actually finished, but I later found that I needed to refactor the code for better supporting other features development. And now I have to reimplement this feature. Development is moving a bit slowly due to my limited time on this project, but I think it won't be too long to release the next version including this feature.

**Comment by CsatiZoltan** [comment]
@chanhx OK, no hurry. Please let me know when it's ready. Thanks.

**Comment by chanhx** [comment]
This feature has been added in the latest v0.5.0, please update the extension and give it a try.

I'm closing this issue. If there are any problems, feel free to reopen or open a new one.

**Comment by CsatiZoltan** [comment]
The interactivity works in the exported HTML file. Thank you!
